model/role.go

```diff
@@ -173,6 +176,23 @@ func IsValidRoleName(roleName string) bool {
 func MakeDefaultRoles() map[string]*Role {
 	roles := make(map[string]*Role)
 
+	roles[CHANNEL_GUEST_ROLE_ID] = &Role{
+		Name:        "channel_guest",
+		DisplayName: "authentication.roles.channel_guest.name",
+		Description: "authentication.roles.channel_guest.description",
+		Permissions: []string{
+			PERMISSION_READ_CHANNEL.Id,
```

Comment:
I have to review this permissions in the Spec and set here exactly what is expected

Reply:
:+1:

Reply:
List looks reasonable to me. Have you reviewed this?

Reply:
The list comes from the specification document.

Reply:
:+1: